js/libs/core.js

```diff
@@ -322,6 +322,15 @@ Fliplet.FormBuilder = (function() {
         'props', 'computed'
       ]), component);
 
+      component.methods.includesComponent = function(container, value) {
+        var returnValue = false;
```

Comment:
Pretty sure this whole function can be rewritten as:

```
return container.indexOf(value) !== -1;
```

Reply:
Also: your function does not seem to be specific to a component or a container, so why name it as such?

Essentially this is like:

```js
function arrayIncludes(arr, value) {

}
```

Reply:
Fixed. I used 'indexOf' in IF block